docs/install_linux.md

```diff
@@ -3,61 +3,89 @@ Welcome to the installation guide for WildBeast on Linux! In this guide, we'll w
+	- A code editor, i.e. [Notepad++](https://notepad-plus-plus.org/) or [Atom](https://atom.io) or [Brackets](http://brackets.io/)
 
 ##Pre-setup
-We higly recommend that you go through a basic Linux server setup before starting this, which includes adding a new user, disabling root login (If you feel so) and adding key authentication for logins. If you have no idea what we mean by this, see the [Digital Ocean guide for initial server setup](https://www.digitalocean.com/community/tutorials/initial-server-setup-with-ubuntu-14-04). When this is complete, you can proceed to the next step.
+We highly recommend that you go through a basic Linux server setup before starting this, which includes adding a new user, disabling root login (If you feel so) and adding key authentication for logins. If you have no idea what we mean by this, see the [Digital Ocean guide for initial server setup](https://www.digitalocean.com/community/tutorials/initial-server-setup-with-ubuntu-14-04). When this is complete, you can proceed to the next step.
 
 ##Installation
 General note: During installation, some administrative tasks will be executed. For these to work (Especially installations), **you need to run these commands as sudo**! They might error if you don't.
 Luckily, the process of executing as sudo isn't that complicated. Just put `sudo` ahead of every install command or other administrative equivalent.
 You will be asked for a password when doing the command, that's all that really is added.
 **Keep this in mind when setting up, so you don't waste our time with errors that stem from lacking permissions!**
-This does of course not apply to you who choose to not do the basic initial server setup and instead run as root.
+While you may think running the following commands as the root user is easy, we advice against it due to the security risks involved when using such a powerful user, instead please use `sudo`.
 ##Installing Node
-First off, we'll install Node.JS, the library WildBeast uses. The current 6.x.x version works fine.
+First off, we'll install Node.JS, the runtime WildBeast uses. The current 6.x.x version works fine.
 ```bash
-wget -qO- https://deb.nodesource.com/setup_6.x | sudo bash -
+sudo apt-get install -y curl
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
 sudo apt-get install -y nodejs
 sudo apt-get install -y build-essential
 ```
 After that is completed, you can check your Node version by executing the following:
 ```bash
-node --version
+node -v
 ```
 The output should then be this or close to that.
 ```bash
-v5.10.1
+v6.9.1
 ```
 ##Retrieving WildBeast
 Now we'll retrieve the WildBeast files via Git. Install it with the following command:
 ```bash
-sudo apt-get install git
+sudo apt-get install -y git
 ```
-When the install completes, clone the WildBeast Git repository.
+When the install completes, clone the WildBeast Git repository and change into the newly created directory.
```

Comment:
The command you suggest uses a parameter for a followup command, which in itself is nice. However, I think that you should remove the `and change into the newly created directory` part to avoid confusion for the end user with duplicate information. Minor thing but still 😄